src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCacheEntry.java

```diff
@@ -65,21 +66,69 @@
     private int responseSize = 0;
     private int subscribers;
     private boolean responseCompleted = false;
-
+    private boolean validationRequired = false;
```

Comment:
Avoid explicitly assigning fields to their natural defaults; all `int` fields default to `0`, all `boolean` fields default to `false`.

Reply:
Done.